src/hash/partition.rs

```diff
@@ -467,7 +468,9 @@ mod private {
                 if table.len() < Self::MIN_TABLE_SIZE {
                     if let Some(i) = self.smallest_table() {
                         for hash in table.drain() {
-                            self.tables[i].insert_unique_unchecked(hash);
+                            unsafe {
+                                self.tables[i].insert_unique_unchecked(hash);
+                            }
```

Comment:
Not actually sure about this one, but I just don't fully understand the invariants here.